bin/MadGraph5_aMCatNLO/cards/production/13TeV/Wgamma_Monophoton_Ptg130_NLO_fxfx/Wgamma_Monophoton_Ptg130_NLO_run_card.dat

```diff
@@ -0,0 +1,149 @@
+# ickkw parameter:                                                     *
+#   0: No merging                                                      *
+#   3: FxFx Merging - WARNING! Applies merging only at the hard-event  *
+#      level. After showering an MLM-type merging should be applied as *
+#      well. See http://amcatnlo.cern.ch/FxFx_merging.htm for details. *
+#   4: UNLOPS merging (with pythia8 only). No interface from within    *
+#      MG5_aMC available, but available in Pythia8.                    *
+#  -1: NNLL+NLO jet-veto computation. See arxiv:1412.8408 [hep-ph].    *
+#***********************************************************************
+  3     =  ickkw
+ #**********************************************************************
+#
+#*********************************************************************** 
+#
+#***********************************************************************
+# BW cutoff (M+/-bwcutoff*Gamma)                                       *
+#***********************************************************************
+  15.0  =  bwcutoff
+ #**********************************************************************
+# Cuts on the jets                                                     *
+# Jet clustering is performed by FastJet.
+# When matching to a parton shower, these generation cuts should be    *
+# considerably softer than the analysis cuts.                          *
+# (more specific cuts can be specified in SubProcesses/cuts.f)         *
+#***********************************************************************
+  1.0   =  jetalgo    ! FastJet jet algorithm (1=kT, 0=C/A, -1=anti-kT)
+  0.7   =  jetradius  ! The radius parameter for the jet algorithm
+  10.0  =  ptj        ! Min jet transverse momentum
+  -1.0  =  etaj       ! Max jet abs(pseudo-rap) (a value .lt.0 means no c
+#***********************************************************************
+# Cuts on the charged leptons (e+, e-, mu+, mu-, tau+ and tau-)        *
+# (more specific gen cuts can be specified in SubProcesses/cuts.f)     *
+#***********************************************************************
+  0.0   =  ptl      ! Min lepton transverse momentum
+  -1.0  =  etal     ! Max lepton abs(pseudo-rap) (a value .lt.0 means no
+  0.0   =  drll     ! Min distance between opposite sign lepton pairs
+  0.0   =  drll_sf  ! Min distance between opp. sign same-flavor lepton pa
+  0.0   =  mll      ! Min inv. mass of all opposite sign lepton pairs
+  30.0  =  mll_sf   ! Min inv. mass of all opp. sign same-flavor lepton p
+#***********************************************************************
+# Photon-isolation cuts, according to hep-ph/9801442                   *
+# When ptgmin=0, all the other parameters are ignored                  *
+#***********************************************************************
+  130.0 =  ptgmin     ! Min photon transverse momentum
+  2.6   =  etagamma   ! Max photon abs(pseudo-rap)
```

Comment:
photon eta is 2.6 but above 3.6. what is the cut at analysis level?

Reply:
We use barrel eta at analysis level

Reply:
closing this request as we have opened https://github.com/cms-sw/genproductions/pull/2800